autoload/leaderf/python/leaderf/asyncExecutor.py

```diff
@@ -33,7 +33,7 @@ def _readerThread(self, fd, queue):
         finally:
             queue.put(None)
 
-    def execute(self, cmd, encoding=None, cleanup=None, env=None, raise_except=True):
+    def execute(self, cmd, encoding=None, cleanup=None, env=None, raise_except=True, format_line=lambda x: x):
```

Comment:
`format_line=None`, then `yield format_line(line.rstrip(b"\r\n").decode()) if format_line else line.rstrip(b"\r\n").decode()`
Maybe it is a little faster.

Reply:
It looks better that way.
I fixed 6582dfe0041e7e01fe80a0d6d1b991f93a725798.